scripts/benchmarks.py

```diff
@@ -109,6 +109,17 @@ def accept(seed, testdata, value):
     return accept
 
 
+def lower_bound(seed, testdata, value):
+    rnd = random.Random(seed)
+    for b in testdata.buffer:
+        c = rnd.randint(0, 255)
+        if c < b:
+            return True
+        if c > b:
+            return False
+    return True
```

Comment:
I can follow the logic here, but I’m not entirely sure what it’s telling me. Perhaps a docstring?

Also, this is a personal preference, but I like to have an underscore prefix for unused arguments, i.e.,

```python
def lower_bound(seed, testdata, _value)
```

but that’s just a suggestion.